InfiniteRecharge/src/main/java/frc/robot/config/Config.java

```diff
@@ -5,6 +5,17 @@
 
 public class Config {
 
+    //////////////////////////////////////////////////////////////////////////////
+    // Subsystem Enablers
+    
+    public boolean ENABLE_SHOOTER_SUBSYSTEM = true;
+    public boolean ENABLE_BALL_MANAGEMENT_SUBSYSTEM = true;
+    public boolean ENABLE_DRIVE_SUBSYSTEM = true;
+    public boolean ENABLE_CLIMB_SUBSYSTEM = true;
+    public boolean ENABLE_INTAKE_SUBSYSTEM = true;
+    public boolean ENABLE_SPINNYBOI_SUBSYSTEM = true;

```

Comment:
These are good at the top of the file, but we should use them like we do motor ids, meaning as part of each system's config file. I.e.

```
if (config.shooter.enabled) {
...
}
```